addons/flannel/0.21.5/install.sh

```diff
@@ -329,4 +329,151 @@ function flannel_already_applied() {
 
     flannel_ready_spinner
     check_network
+    check_udp_port
+}
+
+function flannel_post_init() {
+    check_udp_port
+}
+
+function check_udp_port() {
+    logStep "Checking Flannel UDP Connection over port 8472"
+
+    # Get the number of worker nodes in the cluster
+    # If we have at least 2 nodes we will use affinity to be able to test it across the nodes
+    local num_nodes=$(kubectl get nodes --selector='!node-role.kubernetes.io/master' --no-headers | wc -l)
+
+    echo "Creating Pods to check communication"
```

Comment:
It still now working well as expected, see: 

```
2023-06-03 11:51:41+00:00 No resources found
2023-06-03 11:51:41+00:00 Creating Pods to check communication
2023-06-03 11:51:41+00:00 Create pod to do the test without use podAntiAffinity because has only one node
2023-06-03 11:51:41+00:00 pod/udp-receiver created
2023-06-03 11:51:44+00:00  [|]   [/]   [-]  Creating udp-sender without podAntiAffinity
2023-06-03 11:51:44+00:00 pod/udp-sender created
2023-06-03 11:51:44+00:00 Waiting up to 2 minutes to check pods created to perform the test are successfully running
2023-06-03 11:53:57+00:00  [|]   [/]   [-]   [\]   [|]   [/]   [-]   [\]   [|]   [/]   [-]   [\]   [|]   [/]   [-]   [\]   [|]   [/]   [-]   [\]   [|]   [/]   [-]   [\]   [|]   [/]   [-]   [\]   [|]   [/]   [-]   [\]   [|]   [/]   [-]   [\]   [|]   [/]   [-]   [\]   [|]   [/]   [-]   [\]   [|]   [/]   [-]   [\]   [|]   [/]   [-]   [\]   [|]   [/]   [-]   [\]   [|]   [/]   [-]   [\]   [|]   [/]   [-]   [\]   [|]   [/]   [-]   [\]   [|]   [/]   [-]   [\]   [|]   [/]   [-]   [\]   [|]   [/]   [-]   [\]   [|]   [/]   [-]   [\]   [|]   [/]   [-]   [\]   [|]   [/]   [-]   [\]   [|]   [/]   [-]   [\]   [|]   [/]   [-]   [\]   [|]   [/]   [-]   [\]   [|]   [/]   [-]   [\]   [|]   [/]   [-]   [\]   [|]   [/]   [-]   [\]   [|]   [/]   [-]   [\]  Pods to check UDP communication with Flannel did not become Ready within the expected time.
2023-06-03 11:53:57+00:00 Waiting up to 2 minutes to check if the message will be received
2023-06-03 11:56:04+00:00  [|]   [/]   [-]   [\]   [|]   [/]   [-]   [\]   [|]   [/]   [-]   [\]   [|]   [/]   [-]   [\]   [|]   [/]   [-]   [\]   [|]   [/]   [-]   [\]   [|]   [/]   [-]   [\]   [|]   [/]   [-]   [\]   [|]   [/]   [-]   [\]   [|]   [/]   [-]   [\]   [|]   [/]   [-]   [\]   [|]   [/]   [-]   [\]   [|]   [/]   [-]   [\]   [|]   [/]   [-]   [\]   [|]   [/]   [-]   [\]   [|]   [/]   [-]   [\]   [|]   [/]   [-]   [\]   [|]   [/]   [-]   [\]   [|]   [/]   [-]   [\]   [|]   [/]   [-]   [\]   [|]   [/]   [-]   [\]   [|]   [/]   [-]   [\]   [|]   [/]   [-]   [\]   [|]   [/]   [-]   [\]   [|]   [/]   [-]   [\]   [|]   [/]   [-]   [\]   [|]   [/]   [-]   [\]   [|]   [/]   [-]   [\]   [|]   [/]   [-]   [\]   [|]   [/]   [-]   [\]  Unable to check that Flannel UDP communication is working correctly
2023-06-03 11:56:04+00:00 Clean up the pods created to do the check
2023-06-03 11:56:04+00:00 pod "udp-receiver" deleted
2023-06-03 11:56:04+00:00 pod "udp-sender" deleted
2023-06-03 11:56:04+00:00 ⚙  Checking proxy configuration with Containerd
2023-06-03 11:56:04+00:00 Skipping test. No HTTP proxy configuration found.
2023-06-03 11:56:04+00:00
```